docs/components/layout/layout.module.css

```diff
@@ -7,6 +7,7 @@
     "sidebar content";
   height: 100vh;
   background-color: var(--bg-primary);
+  overflow-x: hidden;
```

Comment:
Jeg tror det er dumt å sette `overflow-x: hidden` her. Om vi ved en feil ender opp med for bredt innhold på en side er det bedre om brukeren kan side-scrolle for å se alt innholdet enn at det skal croppes vekk og være utilgjengelig.